src/osd/PG.cc

```diff
@@ -3730,16 +3732,15 @@ void PG::_scan_snaps(ScrubMap &smap)
  */
 int PG::build_scrub_map_chunk(
   ScrubMap &map,
-  hobject_t start, hobject_t end, bool deep, uint32_t seed,
-  ThreadPool::TPHandle &handle)
+  hobject_t start, hobject_t end, uint32_t seed,
+  ThreadPool::TPHandle &handle, vector<hobject_t> &ls)
```

Comment:
Nit: by convention I've been passing handle last.  Also, per the style guide, ls should probably be passed by pointer since it's an out param?